src/pages/artist/components/Overview.js

```diff
@@ -1,11 +1,142 @@
 import React from 'react'
+import Albums from '../../../molecules/Albums'
+import Heading from '../../../atoms/Heading'
+import { Box } from 'atomic-layout'
 
-function Overview() {
-  return (
-    <div>
-      <p>Overview</p>
-    </div>
-  )
+const albums = [
+  {
+    src: 'https://satyr.io/400x400',
+    nameAlbum: 'Hybrid Theory Live Around the World',
```

Comment:
`nameAlbum` --> `name`.

You are already in `albums` list, so it makes sense: `albums[i].name`.

Reply:
This one is still relevant. Please improve the key names :)